handbook/python-dataplugins/python-dataplugins.md

```diff
@@ -0,0 +1,264 @@
+# Python DataPlugins
+
+Create a DataPlugin to load, to register, or to search your own file formats in LabVIEW or DIAdem, or to index and to browse your own file formats with SystemLink DataFinder.
+
+DataPlugins can be created by using C++, VBS, LabVIEW or Python.

```

Comment:
C++ DataPlugin SDK is not (yet) publicly available. Not sure if we should list it here, so as not to give the impression that customers can create C++ DataPlugins.

Reply:
I took the text from this publicly available document: https://www.ni.com/en-us/support/documentation/supplemental/04/dataplugins-programmer-s-reference-guide.html